osu.Game/Graphics/Containers/SectionsContainer.cs

```diff
@@ -20,6 +22,7 @@ public class SectionsContainer<T> : Container<T>
         where T : Drawable
     {
         public Bindable<T> SelectedSection { get; } = new Bindable<T>();
+        private Drawable externallyScrolledTo;
```

Comment:
I think this needs a better name. Every occurrence of this I have to think about what this actually is. Maybe `lastClickedSection`?

Reply:
If the roundabout way ``ScrollTo(section)`` is triggered in user profile overlay is by clicking the tab title, then sure. Otherwise naming it ``lastClickedSection`` would be wrong, because it can be triggered by means other than clicking buttons.

Reply:
I'm even more confused now, admittedly. What exactly causes an "external scroll"?

Reply:
> What exactly causes an "external scroll"?

Usages of ``ScrollTo()``. 
![2020-12-05 16_34_27-Window](https://user-images.githubusercontent.com/11320483/101245760-c68f7d00-3717-11eb-9756-dedda99344ef.png)

For settings, it is clicking the sidebar button, but for user profile it's some complicated change in value, which is probably triggered by clicking on the tab titles.
![osu_2020-12-05_16-37-05](https://user-images.githubusercontent.com/11320483/101245814-21c16f80-3718-11eb-89dc-f7372e914ddc.png)

Reply:
Do you mean the carousel? User profile's "external scroll" logic is dead simple as far as I can see.

Reply:
https://github.com/ppy/osu/blob/a40a16955c319a052c2d604088682cbb01aa0bc3/osu.Game/Overlays/UserProfileOverlay.cs#L114

It's specifically this line that triggers the ``ScrollTo`` when the current tab changes. But I don't know much about how ``TabControl`` classes work and if current tab can only be changed by clicking on the tab title.

Reply:
Yeah, that's just a click on a tab.